geoalchemy2/shape.py

```diff
@@ -24,14 +24,19 @@ def to_shape(element):
         lake = Session.query(Lake).get(1)
         polygon = to_shape(lake.geom)
     """
-    assert isinstance(element, (WKBElement, WKTElement))
+    assert isinstance(element, (WKBElement, EWKBElement, WKTElement))
     if isinstance(element, WKBElement):
         return shapely.wkb.loads(bytes(element.data))
+    elif isinstance(element, EWKBElement):
+        # This tells shapely to include SRIDs
+        from shapely import geos
+        geos.WKBWriter.defaults['include_srid'] = True
```

Comment:
It's also strange that you configure `WKBWriter` while `to_shape` uses `shapely.wkb.loads`, which is about reading. No?